### DISCUSSION TITLE
2.8.0.98 Version of reMarkable having same issue as 2.6

### DISCUSSION THREAD

**Discussion by mysteriousmartel** [bordaigorl/rmview#81] [Q&A] [answered]
Hi!

This morning my reMarkable updated to the version in the title and it no longer connects to rmview, despite downloading the app using the 2.6 version method. I think things are breaking down for this new version and wanted to bring it up. And advice on how to fix this would be great, thanks!

**Comment by bordaigorl** [comment, marked as answer]
The `2.6` branch has been reported as only working for some people: I do not own a RM2 so I cannot really debug it (that's why it has not been merged in master).
The issue is that every update wipes out the library folder so you'd need to add the missing library again.
You should be able to fix this by following the instructions in the [wiki](https://github.com/bordaigorl/rmview/wiki/Installing-missing-libcrypto-on-RM2-v2.6-or-above)

**Reply by mysteriousmartel** [reply]
Thanks! I'm working on trying to download the library again now, but even though I have the library file on my PC, the scp and ssh commands that are listed in the directions cannot find the file on my computer. Not sure why I didn't have this issue before, but any follow up help would be great!

**Reply by bordaigorl** [reply]
the commands on the wiki assume you are in the directory where they are stored (e.g. `~\Downloads`).
If you are not, you will need to change directory with `cd ~\Downloads` or whatever makes sense in your system

**Reply by stheid** [reply]
I can confirm, that at least on linux the 2.8 works completely fine when manually adding the libcrypto to /usr/lib as statet in the wiki.

**Reply by mysteriousmartel** [reply]
Thanks! This is all correct. I would like to add that, at least for windows, you may be forced to re-enter the host key in known_hosts. A guide for that can be found [here](https://serverfault.com/questions/321167/add-correct-host-key-in-known-hosts-multiple-ssh-host-keys-per-hostname)
